### PR TITLE
CZECH Translations, small update

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: po/cs.po [hunk 1]
```diff
@@ -5580,7 +5581,7 @@ msgid "client capability negotiation"
 msgstr "%s%s: schopnost klienta, vyžaduji: sasl"
 
 msgid "ls || list || req|ack [<capability> [<capability>...]] || clear || end"
-msgstr ""
+msgstr "s || list || req|ack [<schopnost> [<schopnost>...]] || clear || end"
```
Comment (flashcode) [inline po/cs.po]: Typo here: missing "l"

File: po/cs.po [hunk 2]
```diff
@@ -10813,9 +10814,9 @@ msgstr "%s%s: nedostatek paměti pro nový xfer"
 msgid "%s%s: unable to interpret address: error %d %s"
 msgstr "%s%s: nemohu vytvořit rouru"
 
-#, fuzzy, c-format
+#, c-format
 msgid "%s%s: hashing error"
-msgstr "%s%s: chyba inicializace TLS"
+msgstr "%s%s:  chyba hašování"
```
Comment (flashcode) [inline po/cs.po]: Extra space here?

**Comment by mimi1vx** [comment]
fixed, thx for review

**Comment by flashcode** [comment]
Merged.

**Pull request closed without merging**